sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -9,13 +9,13 @@ EGIT_REPO_URI="http://llvm.org/git/libcxx.git
 	https://github.com/llvm-mirror/libcxx.git"
 CMAKE_MIN_VERSION=3.4.3
 
-[ "${PV%9999}" != "${PV}" ] && SCM="git-r3" || SCM=""
+[[ ${PV} == 9999 ]] && SCM="git-r3" || SCM=""
```

Comment:
> But you don't, and you instead rely on poorly reimplementing C in bash. You're not dealing with compiled code, so you also need to count on increased lexing complexity, word splitting, 

Had you tried the example I mentioned, you would have realized that all these are obviously negligible.

> It has to because the argument to % is a pattern.

Good argument here. This really is bash sucking here: same type of pattern with dash is instant...